docs/guides/qpu-information.mdx

```diff
@@ -37,75 +41,68 @@ The revision version number will increment for fixes that do not break the exist
 *   **2Q error (best)** - The lowest two-qubit (2Q) error on any edge of the device as measured from isolated 2Q randomized benchmarking.
 *   **2Q error (layered)** - Average error per layered gate (EPLG) in a chain of 100 qubits. Average EPLG measures the average gate error in a layered chain of $N$ qubits ($N$=100 here). It is derived from a similar quantity known as the layer fidelity (LF) where EPLG$_{100}$ = 4/5(1-LF$^{\frac{1}{99}}$) and layer fidelity is the process fidelity of the layered chain of $N$ qubits. For details, see the paper [Benchmarking quantum processor performance at scale](https://arxiv.org/abs/2311.05933). Note that in the paper EPLG is defined for process error, but for consistency with the individually reported gate errors here it is quoted for average gate error, thus the factor of 4/5.
 *   **CLOPS** (or CLOPS_h) - Circuit layer operations per second, is a measure of how many layers of a 100x100 circuit (hardware-aware circuit) a QPU (quantum processing unit) can execute per unit of time.
 *   **Status** - The QPU status.
 *   **Region** - Location of the data center where your data and experiments will be hosted and processed.
-*   **Total pending jobs** - The total number of jobs that you have submitted to this QPU.
+* **QPU version** - The version number of a QPU in the form `major.minor.revision`. See [QPU versioning](#qpu-versioning) for details on how this number is assigned.
 *   **Processor type** - Reflects the [topology](/docs/guides/qpu-information) and indicates the approximate qubit count.
+* **Basis gates** <span id="native-gates-on-platform"></span> - Each [processor family](/docs/guides/processor-types) has a native gate set. By default, the QPUs in each family only support running the gates and operations in the native gate set. Thus, every gate in the circuit must be translated (by the transpiler) to the elements of this set. Note that the non-unitary operations are not listed here; use the [method in Qiskit](/docs/guides/get-qpu-information#native-gates-with-qiskit) to see all native gates and operations for a QPU. See a list of all native gates in [this table](#native-gates).
+* **Total pending jobs** - The total number of jobs that you have submitted to this QPU.
+* **Median CZ error** (Heron)
+* **Median ECR error** (Eagle)
+* **Median SX error**
+* **Median readout error**
+* **Median T1** (Relaxation Time) - The T1 time represents the average duration a qubit remains in its excited state $|1\rangle$ before decaying to its ground state $|0\rangle$ due to energy relaxation. This parameter is used to characterize the qubit's energy relaxation behavior, and is expressed in units of seconds (s).
+* **Median T2** (Dephasing Time) - The T2 time denotes the timescale over which a qubit maintains phase coherence of a superposition between the $|0\rangle$ and $|1\rangle$ states. It accounts for both energy relaxation and pure dephasing processes, providing insight into the qubit's coherence properties.
```

Comment:
Add that we report the T2 from a Hahn echo sequence.

Reply:
```suggestion
* **Median T2** (Dephasing Time) - The T2 time denotes the timescale over which a qubit maintains phase coherence of a superposition between the $|0\rangle$ and $|1\rangle$ states. It accounts for both energy relaxation and pure dephasing processes, providing insight into the qubit's coherence properties. We report the T2 from a Hahn echo sequence.
```